app/retail/views.py

```diff
@@ -1179,6 +1183,9 @@ def create_status_update(request):
             if kwargs['activity_type'] == 'wall_post':
                 if 'Email Grant Funders' in activity.metadata.get('ask'):
                     grant_update_email_task.delay(activity.pk)
+                if 'Email Hackathon Participants' in activity.metadata.get('ask'):
+                    participants = HackathonRegistration.objects.filter(hackathon=activity.hackathonevent).all()
+                    hackathon_wall_post_email(activity, participants)
```

Comment:
this will need to be moved to a celery task like the grant one is, or else itll time out for any hack that has more than 10 participants

Reply:
^^^